js/src/vista-sdk/lib/GmodPath.ts

```diff
@@ -313,7 +313,7 @@ export class GmodPath {
     }
 
     private isValidLocation(location?: string): boolean {
-        if (!location) return true;
+        if (!location) return false; //true
```

Comment:
Hmm, `null | undefined` is maybe technically a valid location since its optional?